deploy/crds/pulpproject_v1alpha1_pulp_cr.travis.yaml

```diff
@@ -3,6 +3,11 @@ kind: Pulp
 metadata:
   name: example-pulp
 spec:
+  project: pulp
+  # The image name (repo name) for the pulp image.
+  image: pulp
```

Comment:
These are already defaulted to in:
https://github.com/pulp/pulp-operator/blob/master/playbook.yml

So I do not think they're need here.